### PR TITLE
chore(oms): replace legacy references to omg

### PR DESCRIPTION
**Pull request by matthewhudson**
This also allows fixes the API to verify the validity of `oms.yml` and `oms.yaml` files. [See here](https://github.com/storyscript/hub-api/compare/chore/omg-to-oms?expand=1#diff-9e02e085ea022983b0549ff41ca5b00fR314).

Blocked by a necessary update to https://github.com/storyscript/stories.storyscriptapp.com/pull/28

### CODE REVIEW

File: src/main/java/com/asyncy/apis/hub/Constants.java [hunk 1]
```diff
@@ -34,8 +34,8 @@
 
     public static final String PEM_PRIVATE_KEY;
 
-    public static final String OMG_VALIDATE_URL = "https://stories.storyscriptapp.com/omg/validate";
-    public static final String OPENAPI_VALIDATE_URL = "https://stories.storyscriptapp.com/openapi2omg/convert";
+    public static final String OMS_VALIDATE_URL = "https://stories.storyscriptapp.com/oms/validate";
+    public static final String OPENAPI_VALIDATE_URL = "https://stories.storyscriptapp.com/openapi2oms/convert";
```
Comment (matthewhudson) [inline src/main/java/com/asyncy/apis/hub/Constants.java]: Requires a corresponding change in the hub frontend. I'm not aware of any other consumers of this endpoint.

File: src/main/java/com/asyncy/apis/hub/controllers/HookController.java [hunk 2]
```diff
@@ -146,9 +146,9 @@ public String hook(final HttpServletRequest request,
             logger.info("Successfully updated service_tags for service {} ({})",
                     serviceUuid, repoFullName);
             try {
-                EmailUtil.send(EmailUtil.From.StoryscriptHubOmgUpdateSuccess, contactEmails, emailVars);
+                EmailUtil.send(EmailUtil.From.StoryscriptHubOmsUpdateSuccess, contactEmails, emailVars);
             } catch (UnirestException e) {
-                logger.error("Successfully updated OMG service "
+                logger.error("Successfully updated OMS service "
```
Comment (matthewhudson) [inline src/main/java/com/asyncy/apis/hub/controllers/HookController.java]: Are we aggregating logs anywhere? Is changing the log message going to affect any log monitoring we have in place? @judepereira @williammartin ?

File: src/main/java/com/asyncy/apis/hub/controllers/InstallationsController.java [hunk 3]
```diff
@@ -145,19 +145,28 @@ public Repo getInstallationRepo(final HttpServletRequest request, final HttpServ
...
+                        statusCode = prepareGHRequest(repo.getContentURI(fileName).getPath(), userCreds.getOauthToken()).getStatus();
+                        if (statusCode == 200) {
+                            microserviceYamlRequest = makeGHRequest(repo.getContentURI(fileName).getPath(), userCreds.getOauthToken()).getObject();
+                            break;
+                        }
+                    } 
+
+                    repo.setValidity(statusCode == 200);
+                    if (statusCode == 200) {
                         final String yamlConfig = new String(Base64.getDecoder().decode(microserviceYamlRequest.getString("content").replace("\n", "")), StandardCharsets.UTF_8);
```
Comment (matthewhudson) [inline src/main/java/com/asyncy/apis/hub/controllers/InstallationsController.java]: @JeanBarriere I don't really know Java so please feel free to yell at me 😅

File: src/main/java/com/asyncy/apis/hub/controllers/HookController.java (hunk 2 above)
Comment (judepereira) [reply src/main/java/com/asyncy/apis/hub/controllers/HookController.java]: No, there is no log based monitoring right now. Logs are being aggregated in elasticsearch.

**Comment by JeanBarriere** [comment]
LGTM now :wink:

**Pull request merged**